dotnet-template-samples/17-string-value-transform/README.md

```diff
@@ -0,0 +1,21 @@
+The sample in this folder demonstrates:
```

Comment:
```suggestion
The sample in this folder demonstrates using `derived` parameter to:
```

Just adding as there are other ways to do it using forms on parameters:
Example: https://github.com/dotnet/templating/tree/main/test/Microsoft.TemplateEngine.TestTemplates/test_templates/TemplateWithValueForms

Reply:
Good idea, updated.